spec/features/merchant_items/index_spec.rb

```diff
@@ -16,5 +16,22 @@
       expect(page).to have_content("#{item_3.name}")
       expect(page).to_not have_content("#{item_4.name}")
     end
+
+    it 'displays a disable or enable button' do
+      merchant = Merchant.create!(name: 'Yeti')
+      merchant_2 = Merchant.create!(name: 'Hydroflask')
+      item_1 = merchant.items.create!(name: 'Bottle', unit_price: 10, description: 'H20')
+      item_2 = merchant.items.create!(name: 'Can', unit_price: 5, description: 'Soda')
+      item_3 = merchant.items.create!(name: 'Mug', unit_price: 15, description: 'Coffee')
+      item_4 = merchant_2.items.create!(name: 'Kettle', unit_price: 20, description: 'Tea')
+
+      visit "/merchants/#{merchant.id}/items"
+      expect(page).to have_button("Disable/Enable #{item_1.name}")
+      expect(page).to have_content("available")
+
+      click_button "Disable/Enable #{item_1.name}"
+      expect(current_path).to eq("/merchants/#{merchant.id}/items")
+      expect(page).to have_content("unavailable")
```

Comment:
Just gotta throw in a third click_button here, to enable the item and test that it can be enabled